toolkit/analysis_and_visualization/classification/assets/scripts/pagination.js

```diff
@@ -0,0 +1,1125 @@
+                        } else {
+                            html += "<li class=\"" + classPrefix + "-next J-paginationjs-next\" data-num=\"" + (currentPage + 1) + "\" title=\"Next page\"><a href=\"" + pageLink + "\">" + nextText + "<\/a><\/li>";
+                        }
+                    }
+                    html += "</ul></div>";
+                }
+
+                // Whether to display the navigator
+                if (showNavigator) {
+                    if (formatNavigator) {
+                        formattedString = self.replaceVariables(formatNavigator, {
+                            currentPage: currentPage,
+                            totalPage: totalPage,
+                            totalNumber: totalNumber
+                        });
+                        html += "<div class=\"" + classPrefix + "-nav J-paginationjs-nav\">" + formattedString + "<\/div>";
+                    }
+                }
+
+                // Whether to display the Go input
+                if (showGoInput) {
+                    if (formatGoInput) {
+                        formattedString = self.replaceVariables(formatGoInput, {
+                            currentPage: currentPage,
+                            totalPage: totalPage,
+                            totalNumber: totalNumber,
+                            input: goInput
+                        });
+                        html += "<div class=\"" + classPrefix + "-go-input\">" + formattedString + "</div>";
+                    }
+                }
+
+                // Whether to display the Go button
+                if (showGoButton) {
+                    if (formatGoButton) {
+                        formattedString = self.replaceVariables(formatGoButton, {
+                            currentPage: currentPage,
+                            totalPage: totalPage,
+                            totalNumber: totalNumber,
+                            button: goButton
+                        });
+                        html += "<div class=\"" + classPrefix + "-go-button\">" + formattedString + "</div>";
+                    }
+                }
+
+                // Whether to display footer
+                if (footer) {
+                    formattedString = self.replaceVariables(footer, {
+                        currentPage: currentPage,
+                        totalPage: totalPage,
+                        totalNumber: totalNumber
+                    });
+                    html += formattedString;
+                }
+
+                return html;
+            },
+
+            // Find totalNumber from the remote response
+            // Only available in asynchronous mode
+            findTotalNumberFromRemoteResponse: function (response) {
+                var self = this;
+                self.model.totalNumber = attributes.totalNumberLocator(response);
+            },
+
+            // Go to the specified page
+            go: function (number, callback) {
```

Comment:
Codacy found an issue: [Method 'go' has a complexity of 11. Maximum allowed is 4.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)